lib/Ohmbrewer_Thermostat.h

```diff
@@ -215,6 +227,28 @@ namespace Ohmbrewer {
              */
             Temperature* _targetTemp;
 
+            /**
+             * PID instance for the thermostat
+             */
+            PID* _thermPID;
+
+            //Define Variables we'll be connecting to with PID
+            double setPoint;
+            double input;
+            double output;
+
+            //Define the aggressive and conservative Tuning Parameters for PID
+            double aggKp=4;
+            double aggKi=0.2;
+            double aggKd=1;
+            double consKp=1;
+            double consKi=0.05;
+            double consKd=0.25;
+
+            //PID windowing variables
+            int windowSize = 5000;
+            unsigned long windowStartTime;
```

Comment:
These PID-related variables should be static const. Otherwise they're associated with particular instances and can be modified during operation (which I don't think we want to be able to do at the moment).